dev.mufeed.Wordbook.json

```diff
@@ -0,0 +1,99 @@
+{
+  "app-id": "dev.mufeed.Wordbook",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "45",
+  "sdk": "org.gnome.Sdk",
+  "command": "wordbook",
+  "finish-args": [
+    "--share=network",
+    "--socket=pulseaudio",
+    "--share=ipc",
+    "--device=dri",
+    "--socket=fallback-x11",
+    "--socket=wayland"
+  ],
+  "modules": [
+    {
+      "name": "pcaudiolib",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/espeak-ng/pcaudiolib/archive/1.2.tar.gz",
+          "sha256": "44b9d509b9eac40a0c61585f756d76a7b555f732e8b8ae4a501c8819c59c6619"
+        }
+      ]
+    },
+    {
+      "name": "espeak-ng",
+      "cleanup": [
+        "*.la"
```

Comment:
Move this to a global cleanup. Add `*.a`, `*.la`, `/lib/pkgconfig`, `/include`

Reply:
Hi, should be fixed now!